src/Chart.Bar.js

```diff
@@ -276,7 +276,7 @@
 			helpers.each(this.datasets,function(dataset,datasetIndex){
 				helpers.each(dataset.bars,function(bar,index){
 					if (bar.hasValue()){
-						bar.base = this.scale.endPoint;
+						bar.base = this.scale.zeroPoint;
```

Comment:
This made for some weird behavior on my graph with all positive values.
![screen shot 2015-01-12 at 2 56 53 pm](https://cloud.githubusercontent.com/assets/467487/5712914/46a1bb52-9a6b-11e4-8542-411a06d3acaa.png)